pysyft/types/syft/v1/arg.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package pysyft.types.syft.v1;
+
+import "pysyft/types/syft/v1/pointer_tensor.proto";
+import "pysyft/types/syft/v1/shape.proto";
+import "pysyft/types/syft/v1/virtual_worker.proto";
+import "pysyft/types/torch/v1/tensor.proto";
+import "pysyft/types/torch/v1/parameter.proto";
+
+
+message Arg {
+    oneof arg {
+        bool arg_bool = 1;
+        int32 arg_int = 2;
+        float arg_float = 3;
+        string arg_string = 4;
+
+        // TODO: Decide if we can live without VirtualWorker and just use int32 here
```

Comment:
The VirtualWorker type is needed, because the `force_full_simplification` option can cause the VirtualWorker to be serialized with its tensors.